lnbits/commands.py

```diff
@@ -159,6 +162,42 @@ async def db_versions():
         return await get_dbversions(conn)
 
 
+@db.command("cleanup-wallets")
```

Comment:
```suggestion
@coro
```
use `@coro` and avoid the loop